about/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Civilian Climate Futures Project - About</title>
+
+    <link rel="stylesheet" href="styles.css" />
+
+    <link
+      rel="stylesheet"
+      media="screen"
+      href="https://cdn.jsdelivr.net/gh/aaaakshat/cm-web-fonts@latest/fonts.css"
+      type="text/css"
+    />
+
+    <meta
+      name="description"
+      content="Speculative fiction from a future with the Civilian Climate Corps"
+    />
+    <meta
+      property="og:title"
+      content="Civilian Climate Futures Project - About"
+    />
+    <meta
+      property="og:description"
+      content="Speculative fiction from a future with the Civilian Climate Corps"
+    />
+    <meta
+      property="og:image"
+      content="https://d2w9rnfcy7mm78.cloudfront.net/1501570/original_07a9028e53c1df0f556acaec16b16dd7.jpg?1513050071?bc=1"
+    />
+    <meta property="og:type" content="website" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta property="og:url" content="https://www.cccfutures.com/" />
+    <link rel="canonical" href="https://www.cccfutures.com/" />
+    <link rel="icon" href="ccc_favicon.png" />
+    <link rel="icon" href="ccc_favicon.svg" type="image/svg+xml" />
+    <meta name="theme-color" content="#FAFF00" />
+  </head>
+  <body></body>
```

Comment:
you can put the code here - if you wanna reuse the header from home this should work out of the box.